src/index.html

```diff
@@ -5,9 +5,43 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/index.scss">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div data-qa="card" class="container">
+      <img
+        src="images/imac.jpeg"
+        alt="monitor"
+        class="monitor"
+      >
+
+      <div class="description">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</div>
+
+      <div class="code">Product code: 195434</div>
+
+      <div class="rating">
+        <div class="stars">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+
+        <div class="rewiev">Reviews: 5</div>
+      </div>
+
+      <div class="mark">
+        <div class="price">Price:</div>
+        <div class="sum">$2,199</div>
+      </div>
+
+      <section class="button">
+        <div data-qa="hover" class="buy">BUY</div>
+      </section>
```

Comment:
The tags section and div is redundant here, will be better if you use only tag a here